### PR TITLE
Feature/fcdf 191 lookup

### PR DESCRIPTION
**Pull request by slavko28**

### CODE REVIEW

File: neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            processField(input, lookupData, field, false);
+        }
+
+        final List<String> notFoundList = lookupData.stream()
+            .filter(uuid -> dataService.getUniqueNodeByProperty(lookupColumn, uuid) == null)
+            .collect(Collectors.toList());
+
+        if (!notFoundList.isEmpty()) {
+            notFoundList.forEach(id -> emitter.emitError(
+                new InvalidEntry<>(1, String.format("Not found in MDM. UUID: %s", id), input)));
```
Comment (mykhalchuk-m) [inline neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java]: The error code needs to be documented appropriately so the message doesn't need to be present in response. The message filed just needs to contain the UUID that was not found.

File: neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                List<StructuredRecord> structuredRecordList = input.get(field.getName());
+                processField(structuredRecordList.get(0), lookupData, childField, true);
+            }
+        } else if (field.getSchema().getType().isSimpleType() && isChildRecord) {
+            LOG.info("It's simple type");
+            if (field.getName().equals(lookupColumn)) {
+                lookupData.add(processProperty(input, field));
+            }
+        } else {
+            LOG.warn("Unrecognized field type");
```
Comment (mykhalchuk-m) [inline neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java]: This log is useless as we expect only particular field types to be processed here.

**Review by mykhalchuk-m — CHANGES_REQUESTED** [review]
Also, please add a ticket into backlog to cover the code with unit tests

File: neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java (hunk 1 above)
Comment (slavko28) [reply neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java]: now message contains only UUID that was not found

File: neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java (hunk 2 above)
Comment (slavko28) [reply neo4j-plugins/src/main/java/io/cdap/plugin/transformer/MdmLookupTransformer.java]: removed useless block with log

**Review by mykhalchuk-m — APPROVED** [review]

**Pull request merged**